test/testinput/prepbufr_adpsfc_api.py

```diff
@@ -210,5 +212,24 @@ def test_bufr_to_ioda():
    print("end")
 
 if __name__ == '__main__':
-   test_bufr_to_ioda()
+    parser = argparse.ArgumentParser()
+    description=(
+            'Reads NCEP PREPBUFR formated ADPsurface input files'
+            ' convert into IODA formatted output files. '
+    )
+
+    DATA_PATH = './testinput/gdas.t12z.adpsfc.prepbufr'
+    OUTPUT_PATH = './testrun/prepbufr_adpsfc_api.nc'
+
+    optional = parser.add_argument_group(title='optional arguments')
+    optional.add_argument('-f', '--filename', type=str, default=DATA_PATH,
+                          dest='filename', help='adpsfc file name')
+    optional.add_argument('-o', '--output', type=str, default=OUTPUT_PATH,
+                          dest='output', help='output filename')
+    optional.add_argument('-d', '--date', type=str, default='202108010000',
```

Comment:
@emilyhcliu and @nicholasesposito ,,,, note this default time does not match the `t12z` in the sample file shown in `DATA_PATH` may want to make these consistent

Reply:
Oh, right, I didn't think about this. So the adpsfc file that was already in here was this 12z file (I don't remember from when). I didn't want to change the other results so I kept this file and used the converter on it, but apparently forgot to change the information in the python script. 

Do we prefer to use the 12z file that's already there or the 202108010000 file?